frontend/src/fixtures/usersFixtures.js

```diff
@@ -1,48 +1,54 @@
 const usersFixtures = {
```

Comment:
For fixtures, I simply added the `suspended` field to each of our examples for consistency.